addons/website_slides/static/src/xml/slide_quiz_create.xml

```diff
@@ -0,0 +1,134 @@
+            <span class="d-none d-sm-inline-block">Update</span>
+        </a>
+        <a class="o_wslides_js_quiz_cancel_question btn btn-light border ml-2" role="button">
+            <span class="d-none d-sm-inline-block">Cancel</span>
+        </a>
+    </t>
+
+    <t t-name="slide.quiz.question.view">
+        <div t-attf-class="o_wslides_js_lesson_quiz_question mt-3"
+             t-att-data-question-id="question.id" t-att-data-title="question.question">
+            <div class="row d-flex mb-2 mx-0">
+                <div class="h4">
+                    <small class="text-muted">
+                        <i class="o_wslides_js_quiz_sequence_handler fa fa-bars mr-1" />
+                        <span class="o_wslides_quiz_question_sequence" t-esc="question.sequence"/>.
+                    </small>
+                    <span t-esc="question.question"/>
+                </div>
+                <div class="ml-auto o_wslides_js_quiz_edit_del">
+                    <i class="o_wslides_js_quiz_edit_question fa fa-pencil-square-o p-1" />
+                    <i class="o_wslides_js_quiz_delete_question fa fa-trash p-1" />
+                </div>
+            </div>
+            <div class="list-group">
+                <t t-foreach="question.answer_ids" t-as="answer">
+                    <a t-att-data-answer-id="answer.id" href="#"
+                       t-att-data-text="answer.text_value" t-att-data-is-correct="answer.is_correct"
+                       t-attf-class="o_wslides_quiz_answer list-group-item d-flex align-items-center list-group-item-action">
+                        <label class="my-0 d-flex align-items-center justify-content-center mr-2">
+                            <input type="radio" class="d-none" t-att-name="question.id" t-att-value="answer.id"/>
+                            <i class="fa fa-circle text-400"/>
+                            <i class="fa fa-times-circle text-danger d-none"/>
+                            <i class="fa fa-check-circle text-success d-none"/>
+                        </label>
+                        <span t-esc="answer.text_value"/>
+                    </a>
+                </t>
+            </div>
+        </div>
+    </t>
+
+    <t t-name="slide.quiz.new.quiz.button">
+        <a class="o_wslides_js_quiz_add btn btn-light border ml-2" role="button">
+            <i class="fa fa-plus mr-2"/>
+            <span class="d-none d-sm-inline-block">Add Quiz</span>
+        </a>
+    </t>
+
+    <t t-name="slide.quiz.new.question.button">
+        <a class="o_wslides_js_quiz_add btn btn-light border ml-2" role="button">
+            <i class="fa fa-plus mr-2"/>
+            <span class="d-none d-sm-inline-block">Add Question</span>
+        </a>
+    </t>
+
+    <t t-name="slide.quiz.confirm.deletion">
+        <p>Are you sure you want to delete this question : <strong><t t-esc="widget.question.data('title')"/></strong></p>
```

Comment:
Missing "?" at the end of sentence.

Also, adding to my previous comment, you would pass questionId and questionTitle to your dialog widget, and make it easier here by calling `widget.questionTitle`.

Also², to reduce code, you can do `<strong t-esc="widget.questionTitle" />`